web/content/js/versions.js

```diff
@@ -266,6 +278,14 @@ const navVersions = [
           {
             "title": "Application Centric Deployment",
             "path": "Deploying_Hosts_AppCentric"
+          },
```

Comment:
I think you should move "updating Foreman" as "Updating Katello" below line 229 and drop "Upgrading Foreman" because it's a duplicate.

Maybe also rename 227 to "Upgrading Katello"?

Reply:
Addressed :+1:

Kept both guides at last.